gpMgmt/bin/README.md

```diff
@@ -28,21 +28,21 @@ Where Things Go
 
 List of Management Scripts Written in Bash
 ------------------------------------------
-bin/gpinitsystem        -  Creates a new Cloudberry Database
+bin/gpinitsystem        -  Creates a new Greenplum Database
```

Comment:
Keep the `Cloudberry` unchanged?

Reply:
yes,i should Keep the Cloudberry unchanged